keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -122,8 +123,11 @@ All existing E2E tests for Services should continue to pass.
 
 ### Graduation Criteria
 
-The graduation criteria of this KEP will largely depend on the graduation status of the EndpointSlice API. Once the `terminating` field is added to EndpointSlice API,
-this change in behavior will kick-in as soon as kube-proxy consumes EndpointSlice.
+#### Alpha
+
+* kube-proxy internally tracks the terminating condition of an endpoint.
+* feature is only enabled if the feature gate `EndpointSliceTerminatingCondition` is on.
+* unit tests in kube-proxy.
```

Comment:
Can you detail this a bit - what are the tests aiming to prove?

Reply:
The unit tests are actually outlined in more details in the Test Plan section https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md#unit-tests, this change was really just to say that those should exist before this feature is alpha.